lib/mongodb.js

```diff
@@ -120,6 +120,45 @@ exports.initialize = function initializeDataSource(dataSource, callback) {
   }
 };
 
+exports.encodeMongoDBURL = encodeMongoDBURL;
+
+function encodeMongoDBURL(url) {
+  let encodedUrl = url;
+  const atIndex = encodedUrl.lastIndexOf('@');
+  let protocol = 'mongodb://';
+  const srvProtocol = 'mongodb+srv://';
+
+  if (url.startsWith(srvProtocol)) protocol = srvProtocol;
```

Comment:
Why are you switching from `mongodb+src://` to `mongodb`? Those two protocols are different and there are good reasons why people want to use `mongodb+src://`.